content/docs/user-guide/setup-google-drive-remote.md

```diff
@@ -1,9 +1,61 @@
 # Setup a Google Drive DVC Remote
 
-Follow this guide to setup Google Drive as your <abbr>DVC project</abbr>'s
-[remote storage](/doc/command-reference/remote).
+In this guide we explain the existing alternatives to setup Google Drive
```

Comment:
minor: alternatives -> ways (a bit easier to read, since other wise it's easy to misread as an alternative to Google Drive)

Reply:
we don't explain Basic setup ... let's do a quick start even before URL, or even overview that includes the basic setup (and even authorization) - like very simple a quick? May be very similar to what we have in `dvc remote add` now. And a few links to URLs and "Using your won project" sections?

Reply:
Using "ways".
And I did already include a paragraph starting in "Note that to start using a GDrive remote, you only need..." but OK, copied the basic example from `remote add` now.
64e85917